vllm/distributed/device_communicators/pynccl.py

```diff
@@ -180,8 +179,8 @@ def from_torch(cls, op: ReduceOp) -> 'ncclRedOp_t':
 _c_ncclAllReduce = nccl.ncclAllReduce
 _c_ncclAllReduce.restype = ctypes.c_int
 _c_ncclAllReduce.argtypes = [
-    ctypes.c_void_p, ctypes.c_void_p, ctypes.c_size_t, ncclDataType_t,
-    ncclRedOp_t, ctypes.c_void_p, ctypes.c_void_p
```

Comment:
This is not intuitive. Let's keep `ncclDataType_t` and `ncclRedOp_t` here.

Does mypy understand the call signature of `_c_ncclAllReduce`? I suppose it will just ignore them. And we can leave a comment here, saying that `int` will be automatically converted by `ctypes`.